Filecoin applications improvements (#466)
filecoin-project/filecoin-docs: docs/build/onboard-testnet.md

For small-scale testing of your application, you can use smaller-scale Filecoin networks, such as a [local network](local-devnet.md) or the [Filecoin Nerpa network](../networks/README.md#nerpa-devnet). We currently recommend _all developers_ use the Filecoin testnet in order to receive direct support from the Protocol Labs team and to be eligible for upcoming data storage rewards.

> The Filecoin testnet is a network with parameters and activity levels as close as possible to the expected early state of the Filecoin main network (mainnet). As a developer on the Filecoin testnet, there are some important things you should know about development on the testnet.

Reviewer: ```suggestion
The Filecoin testnet is a network with parameters and activity levels _as close as possible_ to the expected state of the Filecoin mainnet once it launches. There are some important things developers should know before building on the testnet.
```
